public/components/spotlight.js

```diff
@@ -46,6 +47,13 @@ function handleRefresh(event) {
 
             parentTag.innerHTML = renderStation(randomStationObj)
         });
+}
+
+spotlightStation.addEventListener('click', handleSpotlight)
 
+function handleSpotlight(event) {
+    event.preventDefault();
+    console.log(event.target);
```

Comment:
Overall looks good. One observation at line 56 of spotlight.js - could we please remove the console.log?